api/signup.go

```diff
@@ -205,7 +205,7 @@ func (a *API) Signup(w http.ResponseWriter, r *http.Request) error {
 			}
 
 			if cookie != "" && config.Cookie.Duration > 0 {
```

Comment:
Based on the conversation last night with @thorwebdev, we always want to set the cookie and Thor suggested removing this check. Is this check needed?